features/vaultsOverview/pipes/positionsOverviewSummary.ts

```
@@ -168,5 +169,7 @@ export function createPositionsOverviewSummary$(
     })),
     debounceTime(500),
     distinctUntilChanged(isEqual),
+    timeout(1000 * 10),
+    catchError(() => of({ assetsAndPositions: [], percentageOther: zero, totalValueUsd: zero })),
```

Not for this PR, but we should talk more with other fullstacks about swallowing errors like this.